samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs

```diff
@@ -67,27 +67,39 @@ internal static IServiceCollection AddSemanticKernelServices(this IServiceCollec
     }
 
     /// <summary>
-    /// Create the semantic memory with backing memory store.
+    /// Add the semantic memory.
     /// </summary>
-    private static ISemanticTextMemory CreateSemanticTextMemory(this IServiceProvider serviceProvider)
+    private static void AddSemanticTextMemory(this IServiceCollection services)
     {
-        MemoriesStoreOptions config = serviceProvider.GetRequiredService<IOptions<MemoriesStoreOptions>>().Value;
+        MemoriesStoreOptions config = services.BuildServiceProvider().GetRequiredService<IOptions<MemoriesStoreOptions>>().Value;
         switch (config.Type)
         {
             case MemoriesStoreOptions.MemoriesStoreType.Volatile:
-                return new SemanticTextMemory(
-                    new VolatileMemoryStore(),
-                    serviceProvider.GetRequiredService<IOptionsSnapshot<AIServiceOptions>>().Get(AIServiceOptions.EmbeddingPropertyName)
-                        .ToTextEmbeddingsService(logger: serviceProvider.GetRequiredService<ILogger<AIServiceOptions>>()));
+            {
```

Comment:
Braces {} for each "case"?  Yuck!

Feel free to ignore this comment.  Just expressing aesthetic disagreement :)

Reply:
fair. :) I'll remove them for consistency with the rest of the codebase.